roles/openafs_client/tasks/main.yaml

```diff
@@ -13,6 +13,14 @@
     owner: root
     group: root
 
+- name: Check cache directory
+  file:
+    state: directory
+    path: "{{ afs_viceetcdir }}"
```

Comment:
This directory is the client's configuration directory. It is normally created by the packaging. For "rsync" mode (install from source, we need to create it manually). Maybe that's not done in the right place? Let me look.